src/OmniSharp.Script/ScriptContextProvider.cs

```diff
@@ -66,8 +69,9 @@ public ScriptContext CreateScriptContext(ScriptOptions scriptOptions)
             CompilationDependency[] compilationDependencies = null;
             try
             {
+                var allCsxFiles = _fileSystemHelper.GetFiles("**/*.csx").ToArray();
```

Comment:
the same files are already discovered in ScriptProjectSystem; since this is potentially an expensive operation at start up, we should not do it twice. can we pass a list of files here instead?